tests/lib/tools/snaps-state

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+show_help() {
+    echo "usage: make-snap [snap-name]"
+    echo "usage: install-local [snap-name]"
+    echo "usage: install-local-as [snap-name] [dest-name]"
```

Comment:
Since all commands install a local snap could we do it with just:
```
snap-state install [--classic|--devmode--jailmode] [snap-file]
```

Reply:
@zyga thanks for the suggestion, this is the updated help, is it ok?

usage: pack-local <snap-name>
       install-local <snap-name> [OPTIONS]
       install-local-as <snap-name> <dest-name> [OPTIONS]

Available options:
  --devmode --jailmode --classic

Pack and install commands save the packed snap for future uses,
which is reused on the following calls.
The paths for locating the sources of the snaps to either pack or
install are the local path and then 'tests/lib/snaps/'

Reply:
Yeah, it looks good. Thank you!